### ISSUE TITLE
Apply class to li instead of link

### ISSUE THREAD

**Issue by gregasman** [vigetlabs/nav_lynx#11]
Is it possible to apply the class to an <li> tag instead of on the link?  That's more common for bootstrap.  Thanks.

**Comment by bxt** [comment]
Yes, if you use the `wrapper` option, the slected class will be applied to this wrapper tag instead.
